### DISCUSSION TITLE
I am curious if the evaluation results of Langtest only have a pass rate?

### DISCUSSION THREAD

**Discussion by VelikayaScarlet** [JohnSnowLabs/langtest#965] [General]
I am curious if the evaluation results of Langtest only have a pass rate? A dataset like Stereoset has three scores: LMS, SS, and ICAT, but I only found the pass rate in Langtest, and the number of data pieces does not match the Stereoset dataset. Why is this?

**Comment by ArshaanNazir** [comment]
Thank you for your feedback, @VelikayaScarlet. The specific scores you mentioned are indeed unique to that particular dataset, which is why they weren't included in our initial evaluations. We're planning to incorporate these metrics into our future evaluations to provide a more comprehensive analysis. Also we have developed our own test split to ensure a robust evaluation framework tailored to our specific needs and objectives.
